src/ansys/fluent/core/session.py

```diff
@@ -284,27 +284,29 @@ def upload(self, file_name: str, remote_file_name: Optional[str] = None):
         Parameters
         ----------
         file_name : str
-            file name
+            Name of the local file to upload to the server.
         remote_file_name : str, optional
-            remote file name, by default None
+            Name of the file to be created on the server.
+            If omitted, will maintain the file name from parameter ``file_name``.
+            Directory specification is not supported.
```

Comment:
@hpohekar 

This overlaps with yesterday's discussion about the two methods being no-ops if no service is available. I think you are working on that change. Currently, we see that we are concretely calling the PIM service here. So, presumably, that will change and there will no longer be a PIM dependency here.

Looking at the implementations in the two methods